recipe/meta.yaml

```diff
@@ -27,28 +28,31 @@ requirements:
     - charset-normalizer >=2.0.0,<2.1.0
```

Comment:
`charset-normalizer` does not seem to have a maximum pinning restriction
https://github.com/psf/requests/blob/v2.28.0/setup.py#L62

Reply:
The background of why `charset-normalizer` was introduced https://github.com/psf/requests/commit/2ed84f55b22f19a1e1e8eea2e50963dce62052d3
In my opinion, as we do not have enough info about the stability of charset_normaliser 2.1 with a new version of requests, it's better to  leave pinning as it is